src/Network/MQTT/Client.hs

```diff
@@ -139,6 +139,8 @@ data MQTTConfig = MQTTConfig{
   , _password       :: Maybe String -- ^ Optional password (parsed from the URI)
   , _connectTimeout :: Int -- ^ Connection timeout (microseconds)
   , _tlsSettings    :: TLSSettings -- ^ TLS Settings for secure connections
+  , _pingPeriod     :: Int -- ^ Time in seconds between pings
+  , _pingPatience   :: Int -- ^ Time in seconds for which there must be no incoming packets before the broker is considered dead. Should be more than the ping period plus the maximum expected ping round trip time.
```

Comment:
I think the spec says something about this, but I'm not offended by the defaults.

Reply:
Thanks!